components/mercury/sources/new-transaction/new-transaction.js

```diff
@@ -0,0 +1,61 @@
+const mercury = require("../../mercury.app.js");
+
+module.exports = {
+  key: "mercury-new-transaction",
+  name: "New Transaction",
+  description: "Emits an event for each new transaction in an account.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    mercury,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+    account: {
+      type: "string",
+      label: "Account",
+      optional: false,
+      async options() {
```

Comment:
Does this API support pagination?

Reply:
It does for listing transactions, but not for listing accounts. 
https://docs.mercury.com/reference#accounts